shift_detector/utils/column_management.py

```diff
@@ -36,6 +36,13 @@ def is_categorical(col: pd.Series,
     return unique_fraction <= max_unique_fraction
 
 
+def is_binary(col: pd.Series, allow_na=False):
```

Comment:
why do you set the default to false, if you only ever use this function with true?